NAMESPACE

```diff
@@ -27,6 +29,7 @@ export(flag_excluded_observations)
 export(flag_unsupported_traits)
 export(flag_unsupported_values)
 export(format_flowering_months)
+export(format_min_max_as_range)
```

Comment:
We also have the reverse function to separate a range into min/max - that is just used for custom_R_code. I assume that doesn't need to be exported?

Reply:
Any new functions in custom R code do need to be exported, from now on